### ISSUE TITLE
Bug: PCE Module VCE doesn't render all of the Scanlines.

### ISSUE THREAD

**Issue by pceDev16** [pce-devel/mednafenPceDev#6]
Currently the PCE Module doesn't allow you to see all of the scanlines. 

When running with the following flags to attempt to view all of the available scanlines: `mednafen -pce.slstart 0 -pce.slend 239` it will look as follows,
![240pSuite-0005](https://user-images.githubusercontent.com/95596143/145695191-08809fc1-9624-4773-8841-162124b6cd02.png)
Note the *Bottom* scanline number starting at *3 pixels* before the line at the bottom becomes visible. 

The desired behavior is as follows:
![240pSuite-0004](https://user-images.githubusercontent.com/95596143/145695229-dcc1a744-2d5b-470c-b2dd-ae753bf6154f.png)

**Comment by dshadoff** [comment]
In case you are not aware of it, Chris Covell has an excellent test ROM called "Screen Dimension Test" which will likely be useful.
It can be found here (roughly 15th or 16th on the page):
https://www.chrismcovell.com/creations.html

**Comment by pceDev16** [comment]
Yes, I have this and have used it previously. Super handy !

**Comment by pceDev16** [comment]
Here's a snapshot from Mednafen PCE showing the missing 2 scanlines at the bottom and note the 2 extra black scanlines/offset at the top for a 256x240 raster.

![240pSuite_110-0001](https://user-images.githubusercontent.com/95596143/146706194-ebd6a28e-fd4c-438f-854c-e27f932cd1d6.png)

**Comment by pceDev16** [comment]
But here is a Snapshot of Chis' test running at a resolution of 256x240, and everything seems to be there.
 
![Screen_Dimension_Test-0000](https://user-images.githubusercontent.com/95596143/146707270-bb13d551-9d84-4e76-bc13-5984b705fca2.png)

**Comment by pceDev16** [comment]
More clues from a similar investigation on the MiSTer side:
https://github.com/MiSTer-devel/TurboGrafx16_MiSTer/issues/80

**Comment by dshadoff** [comment]
Not sure what you are saying here... Chris' test ROM is more flexible, updates the registers with explicit values, and has vernier-type markings for reading exact line spacing.  The correct way to use the ROM would be comparing original hardware against Mednafen displays with identical settings... is this how you are testing ?

**Comment by pceDev16** [comment]
Yes, I did a quick pass at this comparison with original Hardware on my BMV's and saw some results that are going to require more testing.

**Comment by pceDev16** [comment]
Linking #13 in for tracking as it increases the default number of visible scanlines.

**Comment by pceDev16** [comment]
Here's an image of Chris' ScreenSize Tool on a BVM running off of Real Hardware over RGB SCART
![0DE2EA3B-C81A-4412-B7F6-FF0F69DB2AA1](https://user-images.githubusercontent.com/95596143/147368376-72747464-c0b2-49c3-8c9c-34d8e59aa3fe.jpg)

Note the horizontal overscan hidden in the under scan to the left and right, do we want to enable that behavior by default as well in MDFN @dshadoff ?

**Comment by dshadoff** [comment]
Sorry for the several-month delay...I just noticed your question.

Horizontal overscan (as produced by the console) should be available from the emulator.
I believe that there is already an option in the emulator to "display horizontal overscan area", but I haven't measured whether it is displaying an *accurate amount* of overscan.

I have submitted a separate request to display a rectangle surrounding the "visible area" on a regular monitor - though that might be better served by a series of boxes with the "safe area" in an innermost box (as has been shown on Artemio's discord on various occasions).

My preference would be to show:
a) by default, all that the machine is capable of displaying, with an optional (i.e. hotkey toggle) bounding box showing what portion should be visible.
b) if set by config.ini, the visible portion of the display (which may itself be definable, as different sets show different amounts).
